src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+let li = document.querySelectorAll('li');
+
+function sortList(list) {
+  const sortedList = Array.from(list).sort((a, b) => {
+    const salaryA = parseFloat(a.getAttribute('data-salary'));
+    const salaryB = parseFloat(b.getAttribute('data-salary'));
```

Comment:
```suggestion
   const salaryA = +a.getAttribute('data-salary').slice(1).replaceAll(',', '');
    const salaryB = +b.getAttribute('data-salary').slice(1).replaceAll(',', '');
```
better to keep full number